frontends/verilog/verilog_lexer.h

```diff
@@ -0,0 +1,48 @@
+#ifndef VERILOG_LEXER_H
+#define VERILOG_LEXER_H
+
+#include "kernel/yosys.h"
+#include "frontends/ast/ast.h"
+#include "frontends/verilog/verilog_parser.tab.hh"
+#include <string>
+#include <memory>
+
+#if ! defined(yyFlexLexerOnce)
+#define yyFlexLexer frontend_verilog_yyFlexLexer
+#include <FlexLexer.h>
+#endif
+
+YOSYS_NAMESPACE_BEGIN
+
+namespace VERILOG_FRONTEND {
+	using parser = frontend_verilog_yy::parser;
+	class VerilogLexer : public frontend_verilog_yyFlexLexer {
+		ParseState* extra;
+		ParseMode* mode;
+	public:
+		parser::location_type out_loc; // TODO private?
```

Comment:
```suggestion
		parser::location_type out_loc;
	public:
```
Seems to compile fine